css/global.css

```diff
@@ -6,7 +6,8 @@
 
     --text-color1 : #3b3939;
     --border-color1 : #b4b1b1;
-    --background-color1 : #eaeaec99;
+    --background-color1 : hsla(240, 5%, 92%, 0.6);
+    --background-color2 : hsl(240, 5%, 92%);
     --box-shadow : 1px 1px 10px 1px #3b3f4266;
```

Comment:
if possible try to use same color format, i.e. either hsl or hex.

Reply:
Yes I actually missed to change this. Thanks for pointing this out . Will fix it